.github/ISSUE_TEMPLATE/bug.yml

```
@@ -0,0 +1,58 @@
+name: 🐞 Bug Report
+description: Tell us about something that's not working the way we (probably) intend.
+body:
+  - type: dropdown
+    id: environment
+    attributes:
+      label: Environment
+      description: Where are you using our software?
+      options:
+        - SaaS (https://sentry.io/)
+        - single-tenant (if you know, you know)
+        - self-hosted (`onpremise` deployment)
+    validations:
+      required: true
+  - type: input
+    id: version
+    attributes:
+      label: Version
+      description: If self-hosted, what version are you running?
+    validations:
+      required: false
+  - type: textarea
+    id: repro
+    attributes:
+      label: Steps to Reproduce
+      description: How can we see what you're seeing? Specific is terrific.
+      value: |-
+        1. foo
+        2. bar
+        3. baz
+    validations:
+      required: true
+  - type: input
+    id: expected
+    attributes:
+      label: Expected Result
+    validations:
+      required: true
+  - type: input 
+    id: actual
+    attributes:
+      label: Actual Result
+    validations:
+      required: true
+  - type: textarea
+    id: more-info
+    attributes:
+      label: More Info 
+      description: Logs? Screenshots? Yes, please.
+    validations:
+      required: false
+  - type: markdown
```

Shall we make use of some behavioral psychology: https://www.pnas.org/content/117/13/7103

```suggestion
  - type: checkboxes
    id: veracity
    attributes:
      label: Confirmation
      options:
        - label: I have filled out this form properly
        - label: I have [searched](https://git.io/JcIay) existing issues on this
  - type: markdown
```

---

Overkill. (Also the link is about top vs. bottom of honesty checks ... not, as I would have expected, about whether honesty checks work in the first place.)

---

Oh wait, I just read the "significance" section more carefully and seems like this paper is actually refuting both top and bottom veracity checks 🤯